src/plugins/intel_cpu/src/config.cpp

```diff
@@ -269,10 +271,7 @@ void Config::readProperties(const std::map<std::string, std::string> &prop) {
     if (exclusiveAsyncRequests)  // Exclusive request feature disables the streams
         streamExecutorConfig._streams = 1;
 
-#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
-    // TODO: multi-stream execution has functional issues on ARM target
-    streamExecutorConfig._streams = 1;
-#endif
+    CPU_ARM(streamExecutorConfig._streams = 1);
```

Comment:
what is the reason of this change?
it's temporary code and its refactoring is meaningless. Moreover, it remove reason of such change.

If want to refactor something, you can take any CPU nodes with such platform specific conditions and split it into node itself and executors.